src/cmd/manage_nsfs.js

```diff
@@ -223,38 +223,26 @@ async function add_bucket(data) {
 /** verify_bucket_owner will check if the bucket_owner has an account
  * after it finds one, it returns the account id, otherwise it would throw an error
  * (in case the action is add bucket it also checks that the owner has allow_bucket_creation)
- * @param {string} bucket_owner account name
+ * @param {string} bucket_owner - bucket owner is the account name
```

Comment:
@naveenpaul1, is that change is better? (regarding your comment [here](https://github.com/noobaa/noobaa-core/pull/7801#discussion_r1484038868)).

Reply:
I think it will make more sense if you can add the parameter description in the above method description, adding a description not align with our previous coding style.
`verify_bucket_owner will check if the bucket_owner has an account`
to
`verify_bucket_owner will check if the bucket_owner(same as account name in account schema) has an account`

Reply:
Thanks, I change it to be part of the description (and not inline variable description).